build-support/bin/generate_github_workflows.py

```diff
@@ -57,7 +57,7 @@ def gha_expr(expr: str) -> str:
     return "${{ " + expr + " }}"
 
 
-def hashFiles(path: str) -> str:
+def hash_files(path: str) -> str:
```

Comment:
This was triggering a warning in my IDE, and should never have been camelCase anyway.